spec/features/questions/show_question_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'User see question with answers', '
+  I want to be able see question with answers
+' do
+
+  given(:user) { create(:user) }
+  given(:question) { create(:question, user: user) }
+
+  scenario 'Authenticated user creates question' do
+    sign_in(user)
+
+    visit question_path(question)
+
+    expect(page).to have_content 'MyString MyString MyText Edit | Delete'
```

Comment:
And also, what is "MyString" and "MyText"? How should I guess that these are factory defaults?

It's better to provide such information within tests:

```ruby
create(:question, title: 'My title', body: 'My question')
```

Otherwise we have some kind of _phantom_ variables.